application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -111,7 +113,9 @@ public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
             .flatMap(fetchedUser -> createForumPost(event, fetchedUser))
             .flatMap(user -> dmUser(event.getChannel(), user, event.getGuild()))
             .flatMap(dmSent -> deleteOriginalMessage(event.getJDA(), args.get(2), args.get(1)))
-            .flatMap(deletedOriginalMessage -> event.getHook().sendMessage("Question Transferred"))
+            .flatMap(deletedOriginalMessage -> event.getHook()
+                .sendMessage(String.format("Question transferred to %s",
+                        getHelperForum(event.getJDA()))))
```

Comment:
its sorta crucial for good UX that its the link to the actual thread, not just #questions

Reply:
I did not do that to avoid promoting new questions, in chat. This is only meant for moderator/user of command that action has been completed. OP already gets a ping in thread and a DM about it. So there's no problem navigating to it.

Reply:
Yeah but the moderator perhaps also wants to navigate there? Or would it be hard to add it in the code?